LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2020 Fortmatic Inc.
```

Comment:
We'll need to update this to `Copyright (c) 2020 Magic Labs, Inc.`

Reply:
done! Thank you.